extension/css/relay.css

```diff
@@ -213,42 +164,43 @@ logomark,
   font-family: var(--InterUI);
```

Comment:
This is `var(--InterUI)` but above I believe we prefixed `--Metropolis` with `--relayMetropolis`:

```diff
-  font-family: var(--Metropolis);
+  font-family: var(--relayMetropolis);
```

Not sure if we want to prefix `--relayInterUI` for consistency or not. 🤷